tests/vmi_configuration_test.go

```diff
@@ -546,29 +546,19 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			})
 
 			It("[test_id:5266]should boot to NIC rom if a boot order was set on a network interface", func() {
+				By("Enabling network boot")
+				var bootOrder uint = 1
+				interfaceDeviceWithMasqueradeBinding := libvmi.InterfaceDeviceWithMasqueradeBinding()
+				interfaceDeviceWithMasqueradeBinding.BootOrder = &bootOrder
+
 				By("Creating a VMI with no disk and an explicit network interface")
 				vmi := libvmi.New(
+					libvmi.WithResourceMemory("32Mi"),
 					libvmi.WithNetwork(v1.DefaultPodNetwork()),
-					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithInterface(interfaceDeviceWithMasqueradeBinding),
+					//Enabling BIOS serial output
```

Comment:
The name of the function is good enough, I find this comment unhelpful. It only distracts the reader.

```suggestion
```

Reply:
Done.